spec/models/user_spec.rb

```diff
@@ -0,0 +1,72 @@
+require "rails_helper"
+require "./app/models/user"
+
+describe User do
+
+  context "when all attributes are correct" do
+
+    it "is created" do
+      user = User.create(name: "John Doe", email: "user@gmail.com", password: "secret", uid: "123")
```

Comment:
Consider using `shoulda-matchers` instead to check for validations and invalid records